pkg/osd_cluster_upgrader/upgrader.go

```diff
@@ -525,7 +532,7 @@ func performClusterHealthCheck(c client.Client, metricsClient metrics.Metrics, c
 	if len(ic) > 0 {
 		icQuery = `,alertname!="` + strings.Join(ic, `",alertname!="`) + `"`
 	}
-	healthCheckQuery := `ALERTS{alertstate="firing",severity="critical",namespace=~"^openshift.*|^kube.*|^default$",namespace!="openshift-customer-monitoring"` + icQuery + "}"
+	healthCheckQuery := `ALERTS{alertstate="firing",severity="critical",namespace=~"^openshift.*|^kube.*|^default$",namespace!="openshift-customer-monitoring",pod!~"^elasticsearch-.*",service!="fluentd"` + icQuery + "}"
```

Comment:
I don't think this is enough to catch all the types of logging failures. I've tested this PR with a few scenarios where logging is 'broken' and the upgrader still fails the health check:
 - Logging pods can't schedule
 - Logging pods never become ready
 - Logging volume fills to a warning/critical level
In these scenarios alerts will fire in the "openshift-logging" namespace that are not related to elasticsearch or fluentd pods/services (in the case of the former two alerts, the pod/service is kube-state-metrics). 

Maybe it just needs to ignore the openshift-logging namespace altogether? Thoughts from anyone else?